charts/bpdm/values.yaml

```diff
@@ -0,0 +1,81 @@
+bpdm-gate:
```

Comment:
I know due to limitations of helm currently child of child dependencies are ignored. But I feel for being more clear here we should still disable postgres dependency here by setting enabled: false

Reply:
done